pipelined/bevy_sprite2/src/render/mod.rs

```diff
@@ -204,114 +240,84 @@ pub fn extract_sprites(
             handle: handle.clone_weak(),
         });
     }
-
-    if let Some(mut extracted_sprites_res) = render_world.get_resource_mut::<ExtractedSprites>() {
-        extracted_sprites_res.sprites.extend(extracted_sprites);
-    }
 }
 
 #[repr(C)]
-#[derive(Copy, Clone, Pod, Zeroable)]
-struct SpriteVertex {
-    pub position: [f32; 3],
-    pub uv: [f32; 2],
+#[derive(Copy, Clone, Debug, Pod, Zeroable)]
+struct SpriteInstance {
+    pub transform: [[f32; 4]; 4],
+    pub sprite_size: [f32; 2],
+    pub uv_min: [f32; 2],
+    pub uv_size: [f32; 2],
 }
 
 pub struct SpriteMeta {
```

Comment:
This could use doc strings; it's pub and its use is not immediately clear.

Reply:
Actually, does this even need to be `pub` at all? I'm not sure where external code would want to inspect / modify this.